src/Styleguide/Pages/Artwork/Sidebar.tsx

```diff
@@ -71,10 +75,29 @@ export class Sidebar extends Component<ArtworkSidebarProps> {
     return (
       <SidebarContainer>
         <Artists artists={artwork.artists} />
+
+        {artwork.is_biddable &&
+          artwork.sale_artwork &&
```

Comment:
Not sure if we really need those additional tests since in real life there should not be the case where artwork `is_biddable` but does not have a sale_artwork association. But that does not hurt anything and makes sure the stories are less likely to crash.